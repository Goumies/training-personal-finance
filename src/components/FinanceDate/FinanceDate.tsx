import React from "react";

interface Props {
  date: Date;
}

export const FinanceDate: React.StatelessComponent<Props> = ({ date }: Props) => {
  const dateText: string = date.toDateString();
  return <span className="date">{dateText}</span>;
};
