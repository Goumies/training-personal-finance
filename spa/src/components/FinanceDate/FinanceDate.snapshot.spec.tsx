import React from "react";
import renderer from "react-test-renderer";
import { FinanceDate, Props } from "./FinanceDate";

it("renders correctly", () => {
  const props: Props = {
    date: new Date(2019, 4, 24)
  };
  const tree = renderer.create(<FinanceDate {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
