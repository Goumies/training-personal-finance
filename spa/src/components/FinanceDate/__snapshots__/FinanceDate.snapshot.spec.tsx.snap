// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="date"
>
  2019-5-24
</span>
`;
